src/Create.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import { getSystems, createVideogame } from './utils';
+import classNames from 'classnames';
+import './Create.css';
+
+class Create extends Component {
+    state = {
+        name: '',
+        system: '',
+        played: Boolean,
```

Comment:
When you initialize state here, you should put in the value as the same data type that is going to be there. In this case as a boolean, you would enter an initial value of true or false, depending on how you prefer your data to appear on the page.